src/index.html

```diff
@@ -18,5 +18,66 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="aimg"
+        href="dfrf"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <a
+          class="nav__link is_active"
+          href="1"
+        >
+          Apple
+        </a>
+        <a
+          class="nav__link"
+          href="2"
+        >
+          Samsung
+        </a>
+        <a
+          class="nav__link"
+          href="3"
+        >
+          Smartpfones
+        </a>
+        <a
+          class="nav__link"
+          href="4"
+        >
+          Laptops & Computers
+        </a>
+        <a
+          class="nav__link"
+          href="5"
+        >
+          Gadgets
+        </a>
+        <a
+          class="nav__link"
+          href="6"
+        >
+          Tablets
+        </a>
+        <a
+          class="nav__link"
+          href="7"
+        >
+          Photo
+        </a>
+        <a
+          class="nav__link"
+          href="8"
+        >
+          Video
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading.